src/ensembl/src/content/app/entity-viewer/hooks/useEntityViewerAnalytics.ts

```diff
@@ -157,14 +158,55 @@ const useEntityViewerAnalytics = () => {
     trackDownload({ ...params, category: 'gene_view_proteins_list' });
   };
 
+  const trackXrefsTabSelection = (tabName: string) => {
+    analyticsTracking.trackEvent({
+      category: 'entity_viewer_sidebar',
+      action: 'change_tab',
+      label: tabName
+    });
+  };
+
+  const trackXrefLinkClick = (params: {
+    tabName: string;
+    linkLabel: string;
+  }) => {
+    const sidebarCategoryMap = {
+      [SidebarTabName.OVERVIEW as string]: 'overview',
+      [SidebarTabName.EXTERNAL_REFERENCES as string]: 'external_references'
+    };
+
+    const { tabName, linkLabel } = params;
+    const categoryName = `entity_viewer_sidebar_${sidebarCategoryMap[tabName]}`;
+
+    analyticsTracking.trackEvent({
+      category: categoryName,
+      action: 'external_reference_clicked',
+      label: linkLabel
+    });
+  };
+
+  const trackGeneDownload = (
+    params: Omit<TrackDownloadPayload, 'category' | 'transcriptId'>
+  ) => {
+    trackDownload({
+      ...params,
+      transcriptId: 'all',
+      category: 'entity_viewer_sidebar'
+    });
+  };
+
   return {
     trackTabChange,
     trackFiltersPanelOpen,
     trackAppliedFilters,
     trackAppliedSorting,
     trackProteinInfoToggle,
     trackProteinDownload,
-    trackExternalLinkClickInProteinsList
+    trackExternalLinkClickInProteinsList,
+    trackXrefsTabSelection,
+    trackXrefLinkClick,
```

Comment:
Notice that a function above is called `trackExternalLinkClickInProteinsList`.

- at least within a single file (such as useEntityViewerAnalytics.ts) we should refer to the same concept by the same name: either ExternalLink or XrefLink
- these links exist in multiple places; so the name of the function should reflect where the link has been clicked.

(see also the dev branch here: https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/entity-viewer/hooks/useEntityViewerAnalytics.ts#L211-L212)